Kickstarter-iOS/Features/PaginationExample/PaginationExampleView.swift

```diff
@@ -0,0 +1,82 @@
+import KsApi
+import SwiftUI
+
+private struct PaginationExampleProjectCell: View {
+  let title: String
+  var body: some View {
+    Text(title)
+      .padding(.all, 10)
+  }
+}
+
+private struct PaginationExampleProjectList: View {
+  @Binding var projectIdsAndTitles: [(Int, String)]
```

Comment:
See my note below - the actual display view is just taking simple types. This _could_ take a list of `Project`s, and that's what I did originally, but it got a little annoying - I had to use `Project.template`, and that's marked as private to `KsApi`. So I chose to just keep it to strings-and-ints for now.